components/bsx/Gallery/Item/GalleryItem.vue

```diff
@@ -263,6 +273,17 @@ export default class GalleryItem extends mixins(
     this.$set(this.nft, 'price', unwrapOrDefault(data).toString())
   }
 
+  private async fetchEvents() {
+    const result = await this.$apollo.query({
+      query: itemEvents,
+      client: this.urlPrefix,
+      variables: {
+        id: createTokenId(this.collectionId, this.id),
+      },
+    })
+    this.events = [...this.events, ...result.data.events]
```

Comment:
why we need this `[...this.events, ...result.data.events]` as there is no pagination or something.

How about `this.events = [...result.data.events]` ?

Reply:
Yeah, you are right. There is no need to concat.